source/documentation/administration_guide/topics/Renaming_the_Manager_with_the_Ovirt_Engine_Rename_Tool.adoc

```diff
@@ -55,3 +55,28 @@ This command modifies the FQDN in each self-hosted engine node's local copy of [
 This command modifies the FQDN in the main copy of [filename]`/etc/ovirt-hosted-engine-ha/hosted-engine.conf` on the shared storage domain.
 
 Now, all new and existing self-hosted engine nodes use the new FQDN.
+
+
+[NOTE]
+====
+The oVirt Engine Rename Tool is designed to work only on local machines. Changing the engine name does not automatically update the name on remote Data Warehouse machines. Changing the names on remote DWH machines must be performed manually.
+====
+
+*For remote Data Warehouse deployments, follow these steps on the remote machine:*
+
+. Remove the following PKI files:
++
+`/etc/pki/ovirt-engine/apache-ca.pem`
+`/etc/pki/ovirt-engine/apache-grafana-ca.pem`
+`/etc/pki/ovirt-engine/certs/*`
+`/etc/pki/ovirt-engine/keys/*`
```

Comment:
Not sure why this line is bold in the file preview
![Screenshot from 2022-09-04 17-27-47](https://user-images.githubusercontent.com/64130977/188318620-8c35b59a-047a-44ce-ab20-b8d6883b2cc7.png)

Reply:
a bug in the asciidoc coding - it thinks the asterisks (*) are Bold formatting
I had to add an escape character to fix it